packages/azure-node/src/generators/app-insights/schema.json

```diff
@@ -17,12 +17,14 @@
         "appInsightsKey": {
             "type": "string",
             "description": "The env variable that stores the app insights key.",
-            "alias": "e"
+            "alias": "e",
+            "x-prompt": "What is your application insights key?"
```

Comment:
I think this should be something like `What is your application insights environment variable key`. I also think the `appInsightsKey` naming is a little misleading. @nandorszentpeteri @KarimHmaissi